crest/Assets/Crest/Crest-Examples/Shared/Scripts/SimpleMotion.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Crest Ocean System
```

Comment:
Could it be added to Crest.Examples namespace?

Reply:
done